react/icon-template.js

```diff
@@ -0,0 +1,32 @@
+// Generates Icon components using SVGR.
+// https://react-svgr.com/docs/custom-templates/
+function defaultTemplate(
+  { template },
+  opts,
+  { imports, interfaces, componentName, props, jsx, exports },
```

Comment:
Where is componentName passed from? 
Can we make the icon component Name more specific to icons, to avoid naming conflicts with other components especially when using named imports.
e.g. `Alert` --> `IconAlert` or `Icon.Alert`

Reply:
instead of 
```
import { Alert } from '@massds/mayflower-react/dist/Icon';
// we can't directly use the Icon Alert if there's another component named Alert
```
If we do `IconAlert`
```
import { IconAlert } from '@massds/mayflower-react/dist/Icon';
<IconAlert />
```

or `Icon.Alert`
```
import Icon from '@massds/mayflower-react/dist/Icon';
<Icon.Alert />
```
not sure if we can do something like this with sub component
```
import {Icon.Alert} from '@massds/mayflower-react/dist/Icon';
<Icon.Alert />
```

Reply:
The changing of the naming scheme to `Icon.Alert` or `IconAlert` is also not all that unique and is in use by other icon react component packages too. Cases of uniqueness should use something with Mayflower or Mass in the name, or the aliasing as your first code example shows.

Reply:
I think at least we should ensure uniqueness within our library. If they are using both our Icons and icons from another library, I think chances are low that they will be using the same icon from multiple different libraries. Right now at least I can see a couple of components within our own library are having naming conflicts